integration/script/run-test

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env node
+
+const { argv, env, kill, exit } = require('process');
+const { spawnSync, spawn } = require('child_process');
+const args = require('minimist')(argv.slice(2));
+const path = require('path');
```

Comment:
Do we need this? From the npm description `This is an exact copy of the NodeJS ’path’ module published to the NPM registry.` ??

Reply:
We could `path` from `package.json`, as well as the `child_process`. I just checked the Node.js doc, we could just `require` them.

Reply:
Yeah then we could remove this dependency?

Reply:
Yes, I deleted them in my local branch, will update the PR soon. Thanks for catching this.

Reply:
Updated